clover_api.rb

```diff
@@ -34,6 +34,42 @@ class CloverApi < Roda
     only_json? true
     use_jwt? true
 
+    # Converting rodauth error response to the common error format of the API
+    json_response_body do |hash|
+      # In case of an error, rodauth returns the error in the following format
+      # {
+      #   (required) "error": "There was an error logging in"
+      #   (optional) "field-error": [
+      #     "password",
+      #     "invalid password"
+      #   ]
+      # }
+      if hash["error"]
```

Comment:
nit: Rodauth has `json_response_error?` helper. http://rodauth.jeremyevans.net/rdoc/files/doc/json_rdoc.html
```suggestion
      if hash["error"]
```